src/librustdoc/html/static/js/search.js

```diff
@@ -1879,7 +1935,7 @@ function initSearch(rawSearchIndex) {
              *   d: Array<string>,
              *   q: Array<string>,
              *   i: Array<Number>,
-             *   f: Array<Array<?>>,
+             *   f: Array<0 | Object>,
```

Comment:
Could you declare the type returned from `buildFunctionSearchType` in `externs.js` so you can show it here as well please?